### DISCUSSION TITLE
one-side markets

### DISCUSSION THREAD

**Discussion by EYadegari** [kernc/backtesting.py#668] [General]
Is the logic of this lib compatible with one-side markets?
If yes? How can I use it?

**Comment by kernc** [comment]
You can simply use [`Strategy.buy()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.buy) and [`Strategy.position.close()`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.position)?

**Reply by EYadegari** [reply]
Oh, I used Strategy.sell() instead of Strategy.position.close(). Thank you.
